python/tchannel/messages/call_request_continue.py

```diff
@@ -2,10 +2,48 @@
 
 from .call_request import CallRequestMessage
 from .types import Types
+from .. import rw
+from . import common
 
 
 class CallRequestContinueMessage(CallRequestMessage):
     """Represent a continuation of a call request (across multiple frames)."""
     message_type = Types.CALL_REQ_CONTINUE
 
-    # TODO
+    __slots__ = (
+        'flags',
+        'checksum',
+        'arg_1',
+        'arg_2',
+        'arg_3',
+    )
+
+    def __init__(
+        self,
+        flags=0,
+        checksum=None,
+        arg_1=None,
+        arg_2=None,
+        arg_3=None,
+        args=None
+    ):
+        self.flags = flags
+        if checksum is not None:
+            checksum = common.ChecksumType.standardize(checksum)
+        self.checksum = checksum or \
+            (common.ChecksumType.none, None)
```

Comment:
Wouldn't you rather do this?

    checksum = checksum or (..., ...)
    checksum = [..].standardize(checksum)